EIPS/eip-6366.md

```diff
@@ -8,11 +8,12 @@ status: Draft
 type: Standards Track
 category: ERC
 created: 2022-01-19
+requires: bit_based_permission
 ---
 
 ## Abstract
 
-This EIP offers an alternative to Access Control Lists (ACLs) for granting authorization and enhancing security. Each permission is represented by a single bit in `uint256` from which we can defined up to `256` permissions and `2²⁵⁶` roles. This approach use bitwise operator and bitmask to determine the access right which is much more efficient and flexible than `string` comparison or `keccak()`. We are able to specify the importance of permission based on the bit order.
+This EIP offers an alternative to Access Control Lists (ACLs) for granting authorization and enhancing security. An `uint256` was used to store permission of given address in a ecosystem. Each permission is represented by a single bit in `uint256`, described in [eip-bit_based_permission](./bit_based_permission.md). Bitwise operator and bitmask are used to determine the access right which is much more efficient and flexible than `string` or `keccak256` comparison.
```

Comment:
Using the plurial sound better here `Bitwise operators and bitmasks are used...`